server/graphql/v1/CollectiveInterface.js

```diff
@@ -1997,14 +1997,8 @@ export const OrganizationCollectiveType = new GraphQLObjectType({
       ...CollectiveFields(),
       email: {
         type: GraphQLString,
-        async resolve(orgCollective, args, req) {
-          if (!req.remoteUser) {
-            return null;
-          }
-          return (
-            orgCollective && req.loaders.getOrgDetailsByCollectiveId.load(orgCollective.id).then(user => user.email)
-          );
-        },
```

Comment:
These changes on `email` are the only thing breaking. I've opened a discussion in the only place that depends on it and will rollback if it's problematic: https://github.com/opencollective/opencollective-rest/pull/451